src/components/productdisplayitem.main.less

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+@import './../style/common.less';
+
+.itemdetail-component {
```

Comment:
Stock icon does not have styling, do we want to add that here?

<label for="category_item_availability_VESTRI_WORDMARK_FITTED_HAT_RD"><div><span class="icon"></span>In Stock</div></label>

Reply:
Yes, it's missing, however it seems that we are about to completely revamp product page next week, so I'll just leave it as is.

In case we decide to use the same availability badge, I've styled it in `productlistitem.main.less` as `.itemdetail-availability-state` so it can be copied from there.